assets/js/apitool-methods-ws.js

```diff
@@ -211,6 +211,17 @@ Request('ledger_entry - DirectoryNode', {
   }
 })
 
+Request('ledger_entry - NFT Page', {
+  description: "Returns an NFT Page object in its raw ledger format.",
+  link: "ledger_entry.html#get-offer-object",
+  body: {
+    "id": "example_get_accountroot",
```

Comment:
The ID is an arbitrary string, but a more appropriate one for this example would be:

```suggestion
    "id": "example_get_nft_page",
```